website/docs/d/policy_vni_pool.html.markdown

```diff
@@ -32,5 +32,5 @@ In addition to arguments listed above, the following attributes are exported:
 
 * `description` - The description of the resource.
 * `path` - The NSX path of the policy resource.
-* `start` - The start range of VNI Pool.
-* `end` - The end range of VNI Pool.
+* `start` - The start range of VNI Pool. Minimum: 75001, Maximum: 16777215.
```

Comment:
Since this is readonly, we don't normally mention validations. But it won't hurt :)

Reply:
Ah make sense, removed it.